deploy/olm-catalog/tektoncd-pipeline-operator/0.3.1/tektoncd-pipeline-operator.v0.3.1.clusterserviceversion.yaml

```diff
@@ -216,14 +237,14 @@ spec:
                 - name: WATCH_NAMESPACE
                   valueFrom:
                     fieldRef:
-                      fieldPath: metadata.namespace
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: POD_NAME
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: tektoncd-pipeline-operator
```

Comment:
@nikhil-thomas @siamaksade should be modify this for downstream operator to openshift-pipeline-operator? 
@nikhil-thomas if we did that what are the implications?

Reply:
@nikhil-thomas yes

Reply:
done